setup-cf-kube-dist.sh

```diff
@@ -1,22 +1,24 @@
 #!/bin/bash
 
 target="$1"
-secdir="${2:-../cloudfoundry/secure}"
-secfile="concourse-secrets.yml.gpg"
-secrets="${secdir}/${secfile}"
 
-# EV 'PIPELINE_PREFIX' = prefix to pipeline name for local customization of test
-#           pipelines
+# EV 'PIPELINE_PREFIX' = prefix to pipeline name for local
+#                        customization of test pipelines
 
-if [ ! -f "${secrets}" ]
-then
-    echo -e 1>&2 "$0: Failed to find the secrets file ${secrets}\n\tPlease specify the correct directory holding \"${secfile}\"."
-    exit 1
+if test -n "${CONCOURSE_SECRETS_FILE:-}"; then
```

Comment:
We should report something sensible when it's not set, rather than attempting to run `gpg -d --no-tty "" 2> /dev/null` and hiding the error message.